src/libvirtApi/network.js

```diff
@@ -33,6 +33,21 @@ export function networkActivate({ connectionName, objPath }) {
     return call(connectionName, objPath, 'org.libvirt.Network', 'Create', [], { timeout, type: '' });
 }
 
+export function networkAddStaticHostEntries({ connectionName, networkName, macAddress, ipAddress, parentIndex }) {
+    const opts = { err: "message", environ: ['LC_ALL=C.UTF-8'] };
+    if (connectionName === 'system')
+        opts.superuser = 'try';
+
+    const cmd = [
+        'virsh', '-c', `qemu:///${connectionName}`,
+        'net-update', networkName,
+        'add-last', 'ip-dhcp-host', `<host mac='${macAddress}' ip='${ipAddress}' />`,
```

Comment:
This will not work for IPv6, as with ipv6 there is no such thing as mac address. Instead, we should use "id" or "name" property (you can find examples of such XML [here](https://libvirt.org/formatnetwork.html#examplesPrivate6))
Citing the libvirt documentation:
`The IPv6 host element differs slightly from that for IPv4: there is no mac attribute since a MAC address has no defined meaning in IPv6. Instead, the name attribute is used to identify the host to be assigned the IPv6 address. For DHCPv6, the name is the plain name of the client host sent by the client to the server. Note that this method of assigning a specific IP address can also be used for IPv4 instead of the mac attribute`